src/wirecloud/platform/wiring/utils.py

```diff
@@ -29,38 +29,43 @@
 from wirecloud.platform.plugins import get_operator_api_extensions
 
 
-def has_component(endpoint, component_id, component_type):
-    c_type, c_id, e_name = tuple(endpoint.split('/'))
+def remove_component(component_type, component_id, wiring_status):
```

Comment:
Why are implementing a generic `remove_component` if we are not removing `operators`?